### PR TITLE
Userinfo

### PR DESCRIPTION
**Pull request by c9dong**

### CODE REVIEW

File: src/components/Home/home.js [hunk 1]
```diff
@@ -1,16 +1,93 @@
...
+    this.props.fetchUserInfo();
+  }
+
   render() {
     return (
       <div>
-        // FIXME: Should put stuff like user profile, status etc.
-        // Landing page stuff
-        Profile goes here
+        { _.isEmpty(this.props.userData) ? '' :
```
Comment (jasonf7) [inline src/components/Home/home.js:27]: `this.props.userData ? (...) : ''`

File: src/components/Home/home.js [hunk 2]
```diff
@@ -1,16 +1,93 @@
...
+          <div className={ClassName(s.profileImage)}>
+            <img 
+              src="http://www.destination360.com/north-america/us/california/san-francisco/images/s/golden-gate-bridge.jpg"
+            />
+            <div className={ClassName(s.avatarContainer)}>
+              <Avatar
+                size={50}
+              > {this.props.userData.initial} </Avatar>
+              <h2>{this.props.userData.name}</h2>
+              <p>{'Level ' + this.props.userData.level}</p>
```
Comment (jasonf7) [inline src/components/Home/home.js]: `<p>Level {this.props.userData.level}</p>`

File: src/reducers/users.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import * as types from '../actions/types';
+
+const initialState = {}
+
+function getInitials(firstName, lastName) {
+  var initial = '';
```
Comment (jasonf7) [inline src/reducers/users.js]: es5 pls

File: src/reducers/users.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  }
+  return initial;
+}
+
+const usersReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case types.GET_USER_INFO:
+      action.userData.name = action.userData.firstName + ' ' + action.userData.lastName;
+      action.userData.initial = getInitials(action.userData.firstName, action.userData.lastName);
+      return action.userData;
```
Comment (jasonf7) [inline src/reducers/users.js]: `
return { ... name: <stuff>, initial: <stuff> }
`

**Review by jasonf7 — CHANGES_REQUESTED** [review]

File: src/components/Home/home.js (hunk 1 above)
Comment (c9dong) [reply src/components/Home/home.js:27]: it's not null

**Review by jasonf7 — APPROVED** [review]

**Pull request merged**